backend/requirements.txt

```diff
@@ -12,4 +12,5 @@ pyjwt==2.4.0
 PyYAML==6.0
 requests==2.27.1
 requests_aws4auth==1.1.1
-sqlalchemy==1.3.16
\ No newline at end of file
+sqlalchemy==1.3.24
```

Comment:
[Clarification] - Bug on sqlalchemy and alembic versions. Unrelated to modularization